build.gradle

```diff
@@ -49,6 +49,25 @@ allprojects {
     version rootProject.version
 }
 
+sourceSets {
+    main {
+        groovy {
+            srcDirs = ['src/main/groovy', 'src/main/java']
+        }
+        java {
+            srcDirs = []
+        }
+    }
+    test {
+        groovy {
+            srcDirs = ['src/test/groovy', 'src/test/java']
+        }
+        java {
+            srcDirs = []
+        }
+    }
+}
```

Comment:
This is not good as it essentially moves all the java code to be compiled by the groovy plugin. Errorprone only runs on java code (so this stops all the errorprones). I'd do what gradle-jdks does and make a subproject project called `gradle-consistent-versions-idea` (the jdks one has one called -groovy) and put this code there under the groovy plugin.